<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modbusWin</class>
 <widget class="QWidget" name="modbusWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>440</width>
     <height>219</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="7" column="2">
     <widget class="QLineEdit" name="modbus">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="d508">
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="val"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="d700">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="d500">
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="optype">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="length">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="d520">
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="test_addr">
      <property name="text">
       <string>700</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="d528">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="read">
      <property name="text">
       <string>read</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="modbusSend_2">
      <property name="text">
       <string>TS发送</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="val_2"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="d524">
      <property name="text">
       <string>40</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="d504">
      <property name="text">
       <string>34</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="addr">
      <property name="text">
       <string>500</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="ceil">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="test_val"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="write">
      <property name="text">
       <string>write</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
